packages/melody-idom/__tests__/ComponentSpec.ts

```diff
@@ -80,7 +81,7 @@ describe('component', function() {
             data => {
                 component(Component, 'test', data);
             },
-            { text: 'Hello' },
+            { text: 'Hello' }
```

Comment:
Why are all trailing comma removed in this file?

Reply:
Most probably I had a different config for Prettier on the Melody workspace and I didn't see that.
I'll change it back right away

Reply:
Yes please.

Reply:
I realized that we changed the comma rule to the `es5` because of build process at some point mosty likely due to a CI change. It may not be his fault.